docs/src/appendix/cheatcodes/start_warp.md

```diff
@@ -31,19 +31,51 @@ impl IContractImpl of IContract<ContractState> {
 // ...
 ```
 
-We can use `start_warp` in a test to change the block timestamp for a given contract:
+We can use `start_warp` in a test to change the block timestamp for contracts:
 
 ```rust
-use snforge_std::start_warp;
+use snforge_std::{start_warp, CheatTarget};
 
 #[test]
-fn test_warp() {
+fn test_warp_one() {
     // ...
 
-    start_warp(contract_address, 1000);
+    start_warp(CheatTarget::One(contract_address), 1000);
 
     dispatcher.set_block_timestamp();
     let new_timestamp = dispatcher.get_block_timestamp();
     assert(new_timestamp == 1000, 'Wrong timestamp');
 }
+
+#[test]
```

Comment:
I would just remove those, it is well explained in the example in `cheat_target.md` I think

Reply:
Yes - please do remove those